### PR TITLE
Release Karaf runtime 4.2.3

### PR DESCRIPTION
**Pull request by fpapon**

### CODE REVIEW

File: src/main/webapp/download.html [hunk 1]
```diff
@@ -67,19 +67,19 @@ <h2 class="pb-3 mb-2"><i class="fas fa-download"></i> Download</h2>
           <div class="card flex-md-row mb-4 box-shadow h-md-250">
             <div class="card-body d-flex flex-column align-items-start">
               <strong class="d-inline-block mb-2 text-success"><i class="fas fa-circle"></i> Latest release</strong>
-              <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.2</span></h3>
-              <div class="mb-1 text-muted">December 18, 2018</div>
+              <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.3</span></h3>
+              <div class="mb-1 text-muted">February 7, 2019</div>
```
Comment (jbonofre) [inline src/main/webapp/download.html]: The date is actually February, 11, 2019.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -84,42 +84,42 @@ <h1 class="display-4 text-uppercase">Upgrade to the Enterprise class platform</h
           <div class="carousel-item active">
             <div class="container">
               <div class="carousel-caption">
-                <p><strong>Karaf Cave 4.1.2 has been released! (21/1/19)</strong> - This is an update patch for Apache Cave 4.1.x, containing bug fixes and improvements. (<a href="news.html">Details</a>)</p>
+                <p><strong>Karaf Runtime 4.2.3 has been released! (7/2/19)</strong> - This is an update patch for Apache Karaf 4.2.x, containing bug fixes and improvements. (<a href="news.html">Details</a>)</p>
```
Comment (jbonofre) [inline src/main/webapp/index.html]: Date is actually 11/2/19.

**Review by jbonofre — CHANGES_REQUESTED** [review]

File: src/main/webapp/download.html (hunk 1 above)
Comment (fpapon) [reply src/main/webapp/download.html]: done

File: src/main/webapp/index.html (hunk 2 above)
Comment (fpapon) [reply src/main/webapp/index.html]: done

**Pull request merged**